sdk/storage_datalake/src/operations/path_head.rs

```diff
@@ -98,7 +98,7 @@ impl HeadPathResponse {
             common_storage_response_headers: (&headers).try_into()?,
             etag: etag_from_headers(&headers)?,
             last_modified: last_modified_from_headers(&headers)?,
-            properties: (&headers).try_into()?,
+            properties: (&headers).try_into().ok(),
```

Comment:
I don't understand this change. Can't we try it without? I'll give you a PR.

Reply:
https://github.com/rickrain/azure-sdk-for-rust/pull/2

Reply:
@cataggar, just checking that you saw my comments [here](https://github.com/rickrain/azure-sdk-for-rust/pull/2/commits/ac6fe5f90d09fff444d72c07de7cc28d190947b8#r859143519) and [here](https://github.com/rickrain/azure-sdk-for-rust/pull/2/commits/ac6fe5f90d09fff444d72c07de7cc28d190947b8#r859145288).  I updated my PR to to [remove unused variables](https://github.com/Azure/azure-sdk-for-rust/pull/722/commits/924081078b808e92c5243f8271784c33cb85cee4), but the other change is necessary. Let me know if there's anything else I can do to move this forward.